src/core/requireIfExists.js

```diff
@@ -34,7 +34,7 @@ function requireIfExists(uri) {
             var failedId = err.requireModules && err.requireModules[0];
             if (failedId === uri) {
                 // fake the module, then ensure it is truly loaded
-                requirejs.undef(failedId);
+                window.requirejs.undef(failedId);
                 define(failedId, function() {
                     return null;
                 });
```

Comment:
I think this should be `window.require` as well

Reply:
I'm not sure what was the original reason behind that, so I've just prefixed it

Reply:
I'm talking about this line https://github.com/oat-sa/tao-core-sdk-fe/pull/35/files#diff-09f3f9877b5513c40971deb09bd70af2R41
I think it should be prefixed with window, to avoid webpack parse.